cmd/ipfs/dist/install.sh

```diff
@@ -3,7 +3,9 @@
 # Installation script for ipfs. It tries to move $bin in one of the
 # directories stored in $binpaths.
 
-bin=ipfs
+INSTALL_DIR=$(dirname $0)
+
+bin=$INSTALL_DIR/ipfs
```

Comment:
sorry, one more comment. Put the value here in quotes:
```
bin="$INSTALL_DIR/ipfs"
```

Otherwise any spaces in the `INSTALL_DIR` path will break this